frontend/src/variables.js

```diff
@@ -1,3 +1,4 @@
 export const variables = {
   apiPath: import.meta.env.VITE_API_PATH,
+  rustApiPath: import.meta.env.VITE_RUST_API_PATH,
```

Comment:
Change it to rustApi instead

Reply:
If you want, we could just make it `rustApi` and `pythonApi` to make it easy to understand - That's the logic I went for in https://github.com/AndreasPB/si-mandatory1/pull/10/files

Reply:
I think it's clean. I'm gonna change it to that :D

Reply:
It's now changedto `pythonApi` and `rustApi`.